components/Board.tsx

```diff
@@ -0,0 +1,78 @@
+import styles from "@/styles/Board.module.css";
+import { useState } from "react";
+
+export function Board() {
+  const [traitBoard, setTraitBoard] = useState(false);
+  const handleAddTrait = (type: string) => {
+    if (type === "toxic") {
+      //setToxicType +1
+    }
+    if (type === "psychoactive") {
+      //setPsychoactiveType +1
+    }
+  };
+  const traitPoints = 1;
+  const colonies = 1;
+  const enemies = 1;
+  const toxic = 1;
+  const psychoactive = 1;
+
+  return (
+    <foreignObject x="0" y="0" width="160" height="160">
```

Comment:
Can you convert this with the SVG graphic from Juan Hernandez https://leniolabs.slack.com/archives/C01A4SDDT9T/p1675552904395889
?